scripts/mkcloud

```diff
@@ -230,7 +232,8 @@ function sshrun()
 EOF
     env|grep -e ^debug_ -e ^pre_ -e ^want_ -e ^net_ -e ^nodenumber -e ^clusterconfig | sort >> $mkcconf
 
-    $scp $qa_crowbarsetup $mkcconf root@$adminip:
+    #TODO TH: discuss if the file extension should be part of the variable or not (like it is now)
```

Comment:
I probably agree with @dirkmueller about the `cloud$n/` subdirectory idea.  Proposal schemas don't change too often, so I would expect the corresponding `.yaml` files can mostly be shared between different versions of the same release.  In some cases GM (say) barclamps will have a newer schema than in devel, so they might need different `.yaml` files for the same scenario - but I expect that will happen less often than between product releases.

We should also think about whether using `.yaml.erb` templates makes sense in some cases.